src/environments/environment.prod.ts

```diff
@@ -1,4 +1,4 @@
 export const environment = {
   production: true,
-  apiUrl: "https://dam14n.github.io/miaprova-test/"
+  apiUrl: "http://localhost:3000"
```

Comment:
Prod environment must never be localhost. Remember that localhost refers to your local computer, taking that into consideration PROD environment will never try to communicate with your local computer.

Just roll back this change.